src/views/components/trace/trace-chart-table/trace-container.vue

```diff
@@ -0,0 +1,110 @@
+<template>
```

Comment:
![image](https://user-images.githubusercontent.com/19775780/62281642-b52dc500-b480-11e9-983f-eb43b9c1efdf.png)

warning need fix

Reply:
use <script lang="js"> not <script type="text/javascript"> to fix it